http/http.go

```diff
@@ -431,8 +431,8 @@ func (h *HTTP) mirrorHandler(w http.ResponseWriter, r *http.Request, ctx *Contex
 	}
 
 	if !ctx.IsMirrorlist() {
-		logs.LogDownload(resultRenderer.Type(), status, results, err)
-		if len(mlist) > 0 {
+		logs.LogDownload(resultRenderer.Type(), r.Method, status, results, err)
+		if len(mlist) > 0 && r.Method == "GET" {
```

Comment:
I have the impression that you could tighten it a bit more, and check the `resultRenderer`. In case it's a JSONRenderer, it's not really a download. 

So it would be about adding `&& resultRenderer.Type() == "REDIRECT"` I think

Reply:
Another comment: why not move the check above, at the level of `if !ctx.IsMirrorlist()`? This way, there's no need to modify `logs.LogDownload` at all.

It could be argued that the file `downloads.log` is meant to log downloads, and HEAD requests are not downloads, so shouldn't be logged in this file. I have the impression that it was the original intention. You can always look at HEAD requests in the nginx logs (assuming you run mirrorbits being a proxy).

OTOH it could be argued that the file `downloads.log` was misnamed from the start, and it's useful to log HEAD requests in there, even if it's not a download?

This is just for the sake of discussion, I don't think it matters much.